planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -260,6 +260,8 @@ BehaviorPathPlannerParameters BehaviorPathPlannerNode::getCommonParam()
   p.turn_signal_search_time = declare_parameter<double>("turn_signal_search_time");
   p.turn_signal_shift_length_threshold =
     declare_parameter<double>("turn_signal_shift_length_threshold");
+  p.turn_signal_remaining_shift_length_threshold =
+    declare_parameter<double>("turn_signal_remaining_shift_length_threshold");
```

Comment:
It would be nice to update this parameter in `BehaviorPathPlannerNode::onSetParam` but it is not yet implemented for the other parameters so maybe it is okay to skip it :sweat_smile: